[2025-02-15T00:12:22.989+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:12:18.257496+00:00 [queued]>
[2025-02-15T00:12:23.003+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:12:18.257496+00:00 [queued]>
[2025-02-15T00:12:23.004+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-15T00:12:23.020+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): get_location_dict> on 2025-02-15 00:12:18.257496+00:00
[2025-02-15T00:12:23.027+0000] {standard_task_runner.py:57} INFO - Started process 1016 to run task
[2025-02-15T00:12:23.034+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'get_location_dict', 'manual__2025-02-15T00:12:18.257496+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp3h17w95b']
[2025-02-15T00:12:23.037+0000] {standard_task_runner.py:85} INFO - Job 54: Subtask get_location_dict
[2025-02-15T00:12:23.149+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-15T00:12:18.257496+00:00 [running]> on host 2079ceed134c
[2025-02-15T00:12:23.327+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='get_location_dict' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:12:18.257496+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:12:18.257496+00:00'
[2025-02-15T00:12:23.369+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T00:12:23.390+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=get_location_dict, execution_date=20250215T001218, start_date=20250215T001222, end_date=20250215T001223
[2025-02-15T00:12:23.449+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T00:12:23.503+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
